core/DataTable.php

```diff
@@ -591,9 +591,6 @@ public function addDataTable(DataTable $tableToSum)
     public function getRowFromLabel($label)
     {
         $rowId = $this->getRowIdFromLabel($label);
-        if ($rowId instanceof Row) {
```

Comment:
:+1: From the code it looks unnecessary. In fact I think, we could rewrite the code to assume it is a key for DataTable::$rows or ID_SUMMARY_ROW.